Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu/mysql:8.0-20.04_beta
+
+ARG CONFIG_VERSION="0.8.22-1"
+
+RUN set -eux && DEBIAN_FRONTEND=noninteractive \
```

Comment:
Does the `DEBIAN_FRONTEND=noninteractive` picks the correct option (mysql-server-8.0) for mysql-apt-config install?

Reply:
we are installing mysql-shell (mysql-server-8.0 is already part of the upstream image). also, mysql-shell supports multiple versions of mysql